lib/tlDevice/BMD/Linux/platform.h

```diff
@@ -39,7 +39,7 @@ HRESULT GetDeckLinkIterator(IDeckLinkIterator **deckLinkIterator);
 #define dlbool_t	bool
 #define dlstring_t	const char*
 
-// DeckLink String conversion fucntions
+// DeckLink String conversion functions
```

Comment:
Could you revert this change? This is actually third party code that I don't want to modify, which reminds me that I need to move it into the "deps" directory to make that more clear.

Reply:
Done, d3dc829